app/src/main/res/drawable/ic_baseline_location_on_24.xml

```diff
@@ -0,0 +1,10 @@
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
Could this xml file and the other one resembling it be made adaptable , to have only one and change the one value that is different?